spdmlib/src/requester/get_measurements_req.rs

```
@@ -83,6 +83,33 @@ impl<'a> RequesterContext<'a> {
         Ok(writer.used())
     }
 
+    pub fn reset_message_m(&mut self, session_id: Option<u32>) {
+        match session_id {
+            Some(session_id) => {
+                let session = if let Some(s) = self.common.get_session_via_id(session_id) {
+                    s
+                } else {
+                    log::error!("can't find session via session id!");
```

The error is ignored, the previous code will return spdm_result_erre!(EFAULT)

---

Use https://github.com/jyao1/rust-spdm/issues/370 to track error handling.